lib/LedgerSMB/PSGI.pm

```diff
@@ -14,6 +14,10 @@ LedgerSMB::PSGI - PSGI application routines for LedgerSMB
 use strict;
 use warnings;
 
+use PGObject;
+
+BEGIN { PGObject->new_registry('ledgersmb') }
```

Comment:
that's a good idea.  Then we'd need to specify that as a minimum version, I suppose :-)